cmd/cli/cmd/manifest/generate/types/type.go

```diff
@@ -3,9 +3,10 @@ package types
 import (
```

Comment:
Please remove the `alpha` command and `s/manifest-gen/generate` from the **Example** section in line 25